test/lib/next-test-utils.js

```diff
@@ -104,14 +104,19 @@ export function runNextCommand(argv, options = {}) {
     __NEXT_TEST_MODE: 'true',
   }
 
+  const hookArgs = options.hook ? ['-r', options.hook] : []
```

Comment:
It looks like we still need to rename `hookArgs` -> `nodeArgs` here since it was changed in the test